compiler/scopelang/from_desugared.ml

```diff
@@ -130,22 +131,50 @@ let rec translate_expr (ctx : ctx) (e : D.expr) : untyped Ast.expr boxed =
              ScopeVar.Map.add v' e' args')
            args ScopeVar.Map.empty)
       m
-  | EApp { f = EOp { op; tys }, m1; args } ->
+  | EApp { f; tys; args } -> (
+    (* Detuplification of function arguments *)
```

Comment:
You say in your PR description that de-tuplification happens during the translation to `lcalc`, but this is actually during the translation to `scopelang`.

Reply:
Ah indeed you're right, that changed since the initial prototype (fixed in the description)